### PR TITLE
add separate codeowner for split out transformer package

### PR DESCRIPTION
**Pull request by MichaelBelousov**
when splitting out the transformer package from the backend package, I did not add a codeowner for the new path
this fixes that

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

**Comment by MichaelBelousov** [comment]
@ColinKerr I'm going to wait for your explicit approval since this marks you as the secondary code owner

**Review by ColinKerr — APPROVED** [review]

**Comment by MichaelBelousov** [comment]
@mergifyio update

**Pull request merged**
